### DISCUSSION TITLE
401 error for feeds with authorization and long password

### DISCUSSION THREAD

**Discussion by Luensche** [bitfireAT/icsx5#124] [Q&A]
Hello,

I wanted to subscribe to a calendar feed. After I entered my credentials and tried to connect, I got an error message with 401 error code.
When I looked on my password, I saw that it wasn't highlighted completely. So I changed my password on the remote site from 64 signs to a shorter one and now the subscription is working. Is there any password limit in ICSx?

**Comment by ArnyminerZ** [comment]
Theoretically there's no limit for the length of the password neither the username. The authentication is performed through basic auth, which doesn't have any limits. However, your server might be imposing the length limit. Which server are you running?
